packages/next/telemetry/storage.ts

```diff
@@ -41,22 +41,29 @@ type EventBatchShape = {
 }
 
 export class Telemetry {
-  private conf: Conf<any>
+  private conf: Conf<any> | null
   private sessionId: string
   private rawProjectId: string
 
   constructor({ distDir }: { distDir: string }) {
     const storageDirectory = getStorageDirectory(distDir)
 
-    this.conf = new Conf({ projectName: 'nextjs', cwd: storageDirectory })
+    try {
+      // `conf` incorrectly throws a permission error during initialization
+      // instead of waiting for first use. We need to handle it, otherwise the
+      // process may crash.
+      this.conf = new Conf({ projectName: 'nextjs', cwd: storageDirectory })
+    } catch (_) {
+      this.conf = null
```

Comment:
Why don't we mock the conf api so that data is still sent and then put a marker in the initialization so that we can filter out the data if needed.